### ISSUE TITLE
model_name in quicktour page results in a 404 error

### ISSUE THREAD

**Issue by vinayakkgarg** [huggingface/diffusers#9438]
### Describe the bug

The model_name in the Quicktour page (runwayml/stable-diffusion-v1-5) results in a 404 error, indicating that the path is currently unreachable. This is causing the following error when attempting to use the model:
```
HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/api/models/runwayml/stable-diffusion-v1-5
```
It seems the path to this model may have been changed, and the documentation should be updated to reflect the correct model name.

### Reproduction

To reproduce the issue, you can use the following minimal code snippet:
```
from diffusers import StableDiffusionPipeline

model_name = "runwayml/stable-diffusion-v1-5"
pipeline = StableDiffusionPipeline.from_pretrained(model_name)
```

### Logs

```shell
HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/api/models/runwayml/stable-diffusion-v1-5
```

### System Info

diffusers 0.30.2
Python 3.10.13

### Who can help?

@stevhliu

**Comment by vinayakkgarg** [comment]
I'd like to fix this. Please help with the correct path to the model/alternate model_name to use in the quicktour documentation.

**Comment by stevhliu** [comment]
Thanks for reporting, this will be fixed in #9428 :)

**Comment by stevhliu** [comment]
Fixed in #9428
